hvantk/hgc/file_utils.py

```
@@ -0,0 +1,232 @@
+
+    Returns:
+        List[hl.MatrixTable]: A new list of matrix table objects with columns reordered to match the reference.
+
+    Raises:
+        IndexError: If ref_index is out of range for the list of matrix tables.
+
+    Example:
+        sorted_mts = sort_mts_cols([mt1, mt2, mt3], ref_index=0)
+    """
+    if not 0 <= ref_index < len(mts):
+        raise IndexError("ref_index is out of range for the provided list of matrix tables.")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)